tests/unit_tests/test_wave_07.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from swap_meet.item import Item
+from swap_meet.vendor import Vendor
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+def test_newest_by_category():
+    item_a = Clothing(age=2.0)
+    item_b = Decor(age=2.0)
+    item_c = Clothing(age=4.0)
+    item_d = Decor(age=5.0)
+    item_e = Clothing(age=3.0)
+    pedro = Vendor(
+        inventory=[item_a, item_b, item_c, item_d, item_e]
+    )
+
```

Comment:
Hmm, this test is called `test_newest_by_category` but it calls `get_best_by_category`?